ecosystem/typescript/sdk_v2/tests/unit/multi_ed25519.test.ts

```diff
@@ -0,0 +1,105 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+import { Deserializer } from "../../src/bcs/deserializer";
+import { Serializer } from "../../src/bcs/serializer";
+import { Hex } from "../../src/core/hex";
+import { Ed25519PublicKey, Ed25519Signature } from "../../src/crypto/ed25519";
+import { MultiEd25519PublicKey, MultiEd25519Signature } from "../../src/crypto/multi_ed25519";
+
+describe("MultiEd25519", () => {
+  it("public key serializes to bytes correctly", async () => {
+    const publicKey1 = "b9c6ee1630ef3e711144a648db06bbb2284f7274cfbee53ffcee503cc1a49200";
```

Comment:
we use consts in a lot of test files, wonder if we should have a consts/helper.ts file in the tests folder that keeps privateKeys, publicKeys, authKeys, etc and then use the same values in tests

Reply:
Good suggestion!